src/test/scala/com/fulcrumgenomics/testing/UnitSpec.scala

```diff
@@ -49,6 +55,15 @@ trait UnitSpec extends FlatSpec with Matchers {
     val in = SamReaderFactory.make().open(bam.toFile)
     yieldAndThen(in.iterator().toIndexedSeq) { in.safelyClose() }
   }
+
+  /** Generates a command line parser for a class to check that the argument annotations are valid. */
+  protected def checkClpAnnotations[T <: FgBioTool](implicit ct: ClassTag[T], tt: TypeTag[T]): Unit = {
```

Comment:
If you're really ambitious, you could instead just search for all CLPs with the CLP annotation and check those directly.  Then we don't need to add a test for each new CLP for this.